.eslintrc.js

```diff
@@ -44,5 +46,13 @@ module.exports = {
 				]
 			}
 		]
-	}
+	},
+	"overrides": [
+		{
+			"files": ["*api.test.js", "*cache.test.js", "*subscriber.test.js"],
+			"rules": {
+				"no-unused-expressions": "off"
+			}
+		}
+	]
```

Comment:
Remove this override
If something like `expect(isUsed).to.be.false;` will throw, use `expect(isUsed).to.be.false();` instead